domain/cloverclowzer.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "and512556-star",
+        "email": "and512556@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://and512556-star.github.io/cloverclowzer.github.io/"
```

no trailing slashes LOL
```suggestion
        "CNAME": "and512556-star.github.io"
```